### DISCUSSION TITLE
/posts/nvim-treesitter-and-textobjects

### DISCUSSION THREAD

**Discussion by giscus** [josean-dev/sveltekit-blog#3] [General]
# /posts/nvim-treesitter-and-textobjects

How to setup treesitter & treesitter text objects to take code editing in Neovim to the next level.

https://www.josean.com/posts/nvim-treesitter-and-textobjects

**Comment by robocode51** [comment]
Thank you for the detailed instructions and accompanying video.

**Comment by MartinDelille** [comment]
Thank you for your awesome video and blog post : I had trouble understanding neovim lua config and you made it clear as water! Would you add an RSS feed to your blog ?

**Comment by channingwalton** [comment]
Thanks for this, very useful.

**Comment by boston-code-coffee** [comment]
this information is so helpful. 
thank you

**Comment by Dan98x** [comment]
Thanks for this, very useful.
